src/module/item/physical/data.ts

```diff
@@ -42,11 +42,9 @@ interface PhysicalSystemSource extends ItemSystemSource {
         value: string;
     };
     containerId: string | null;
-    preciousMaterial: {
-        value: Exclude<PreciousMaterialType, "dragonhide" | "grisantian-pelt"> | null;
-    };
-    preciousMaterialGrade: {
-        value: PreciousMaterialGrade | null;
+    material: {
```

Comment:
Why is this not ItemMaterialData? The type seems to match completely.

Reply:
I think I had some idea that one or both properties would be optional in source but decided to keep them both present but possibly `null`.